packages/cli/src/util/proposerConfig.ts

```diff
@@ -79,6 +80,9 @@ function parseProposerConfigSection(
       throw Error("(Number.isNaN(Number(gas_limit)) 2");
     }
   }
+  if (boost_factor !== undefined && typeof boost_factor !== "string") {
```

Comment:
Funny how `boost_factor` has type `bigint | undefined` yet we check for a string here. Looks confusing along with string checks on other config fields. Anyway it looks fine for the purpose of this PR.

Reply:
bigint is jsoned as a string as its not number safe